files/trips_ft.md

```diff
@@ -13,3 +13,15 @@ Required Attributes	| Description
 `trip_id`			| ID that uniquely identifies a vehicle trip
 `vehicle_name`		| Name of vehicle type, which is to match a description in [`vehicles_ft.txt`](vehicles_ft.md)
 
+File MAY contain the following attributes:
```

Comment:
is this for individual trips wheras route_stats is for groups of trips? suggest a new file